webclient/app/src/app/commons/relationshipAccordion/RelationshipAccordion.jsx

```diff
@@ -62,98 +47,99 @@ function RelationshipAccordion(props) {
-                                    <MenuItem value={field.fieldName}>{field.fieldName}</MenuItem>
-                                ))}
-                            </Select>
+                                helperText={ t("relationship.sourceField.description", {entityName: relationship.otherEntityName ? relationship.otherEntityName : t("relationship.targetEntity")} )}
+                                regex={RegexConfig.relationship}
+                            />
                         </FormControl>
                     </Grid>
-                    <Grid item sm={6}>
-                        <Typography variant={"h6"} gutterBottom>{t("relationshipAccordion.target")}</Typography>
-                        <FormControl fullWidth className={relationshipAccordionStyles.spacerBottom}>
+                    <Grid item sm={1}>
+                        <Typography gutterBottom>{t("relationship.target")}</Typography>
+                    </Grid>
+                    <Grid item sm={5}>
+                        <FormControl fullWidth>
                             <InputLabel
-                                id="demo-simple-select-label">{t("relationshipAccordion.targetDomain")}</InputLabel>
+                                id="targetEntity"
+                                error={!RegexConfig.entityTitle.test(relationship.otherEntityName)}
+                            >{t("relationship.targetEntity") + "*"}</InputLabel>
                             <Select
-                                labelId="demo-simple-select-label"
-                                id="demo-simple-select"
-                                value={otherEntityName}
-                                label={t("relationshipAccordion.targetDomain")}
+                                error={!RegexConfig.entityTitle.test(relationship.otherEntityName)}
+                                labelId="targetEntity"
+                                id="targetEntitySelect"
+                                value={relationship.otherEntityName}
+                                label={t("relationship.targetEntity") + "*" }
                                 onChange={(event) => editRelationshipProperty("otherEntityName", event.target.value)}
                             >
-                                {prepareEntitiesSelection()?.map(entityName => (
-                                    <MenuItem value={entityName}>{entityName}</MenuItem>
+                                {targetEntities.map(entity => (
+                                    <MenuItem value={entity.name} key={entity.id} >{entity.name}</MenuItem>
                                 ))}
                             </Select>
+                            <FormHelperText id="targetEntityHelperText"
+                            error={!RegexConfig.entityTitle.test(relationship.otherEntityName)}
+                            >{ t("relationship.targetEntity.description") }</FormHelperText>
```

Comment:
I do not think you need this here. The label in the select says the same as the Helper text